src/test/scripts/product_versions.py

```diff
@@ -15,18 +15,25 @@ def get_lts_version(argv):
 		product = 'jira-software'
 	elif product == 'bitbucket':
 		product = 'stash'
-
 	if product in known_supported_version:
```

Comment:
The latest LTS version of Bitbucket (7.6) is not supported by this, will this be taken into account?

Reply:
We have the known_supported_versions defined to handle this case. If LTS version come from this script is less than known_supported_version, then it will be ignored.